tests/behat/features/DefaultContent.feature

```diff
@@ -5,17 +5,22 @@ Feature: Default Content
   I should be able to navigate and see the default content
 
   Scenario: View Content
-  Given I am on the homepage
-  Then I should get a "200" HTTP response
-  Then I should see "Welcome to your site!" in the "content" region
-  Then I should get a "200" HTTP response
-  Then I am on "/resources"
-  Then I should get a "200" HTTP response
-  Then I am on "/research"
-  Then I should get a "200" HTTP response
-  Then I am on "/about"
-  Then I should get a "200" HTTP response
-  Then I am on "/page-not-found"
-  Then I should get a "200" HTTP response
-  Then I am on "/access-denied"
-  Then I should get a "200" HTTP response
+    Given I am on the homepage
+    Then I should get a "200" HTTP response
+    Then I should see "Welcome to your site!" in the "content" region
+    Then I should get a "200" HTTP response
+    Then I am on "/resources"
+    Then I should get a "200" HTTP response
+    Then I am on "/research"
+    Then I should get a "200" HTTP response
+    Then I am on "/about"
+    Then I should get a "200" HTTP response
+    Then I am on "/page-not-found"
+    Then I should get a "200" HTTP response
+    Then I am on "/access-denied"
+    Then I should get a "200" HTTP response
+
+  Scenario: Media
+    Given I am logged in as a user with the "site_manager" role
+    And I am on "/admin/content/media"
+    Then I should see 15 ".media-library-item img" elements
```

Comment:
```suggestion
    Then I should see 1 or more ".media-library-item img" elements
```
Not sure this is the correct step definition but this will save us from having to re-write it later if content is added/removed.

Reply:
what if `15 or more` and we'll need a new step def

Reply:
https://github.com/SU-SWS/linky_clicky/blob/5.x/includes/bootstrap/SWSFeatureContext.php#L164

Reply:
That can be for later. We should harvest a bunch of those SD for caravan.